flutter/lib/src/default_integrations.dart

```diff
@@ -8,6 +8,40 @@ import 'package:sentry/sentry.dart';
 import 'sentry_flutter_options.dart';
 import 'widgets_binding_observer.dart';
 
+class RunZoneGuardedIntegration extends Integration<SentryFlutterOptions> {
+  RunZoneGuardedIntegration(this._runBeforeIntegrations, this._runAfterIntegrations);
+
+  final Function() _runBeforeIntegrations;
```

Comment:
wondering if we should create a typedef for this too, being a `Function` is good because of flexibility, but I see this internal only.